### DISCUSSION TITLE
Blueprint simulation inspired from flask into blacksheep

### DISCUSSION THREAD

**Discussion by Legopapurida** [Neoteroi/BlackSheep#145] [Ideas]
Blueprint concept is the best way to achieve scalability like flask 

- [ ] Blueprint 
- [ ] Nested Blueprint

**Comment by myusko** [comment]
hi @Legopapurida this is an interesting idea, in the future we can consider the idea.

At this time I recommend you to check [BlackSheep's controllers](https://www.neoteroi.dev/blacksheep/controllers/), probably this approach will be suitable for your use-cases.

Thanks,
Mykhailo

**Comment by mavwolverine** [comment]
I second this feature. 
`Controllers` is a good feature too. 

But extensibility is an important feature in big projects.

**Comment by mavwolverine** [comment]
@myusko question: can you please provide an example to handle nested APIController?

`/org/1/users/1`
